tests/headers/objc_template.h

```diff
@@ -0,0 +1,6 @@
+// bindgen-flags: --objc-extern-crate -- -x objective-c
+// bindgen-osx-only
+
+@interface Foo<__covariant ObjectType>
```

Comment:
heh yeah, they are a tack on, much like java generics, so it should always be valid to just use id instead, just with less guarantees. 

I suppose those might of been added to ease with the Swift-interop, there are other "hints" in the headers these days to help with that, like 'instancetype' (~Self), 'nullable' (Option opportunity)